### PR TITLE
ZBUG-1885:- Unable to scroll message header in the preview window

### PR DESCRIPTION
**Pull request by madhavdhuppe**

### CODE REVIEW

File: WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ function() {
 	if (this._objectManager && this._objectManager.reset) {
 		this._objectManager.reset();
 	}
-	this.setScrollWithIframe(this._scrollWithIframe ? DwtControl.CLIP : DwtControl.SCROLL);
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: this was added to fix https://github.com/Zimbra/zm-web-client/commit/0caad4f1bc2e29856b0148bd33993bb9ee08ba6b
curious to know how it's working now

Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: we added alternative solution in https://github.com/Zimbra/zm-web-client/pull/594 
which is much better that this
1) no additional blank space
2)supports multiple mimes.

Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: so decided to remove this line of code. as we don't need this anymore

Comment (silentsakky) [reply WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: if that was the case, then we should have removed this code in that same PR only, let's not have not needed code in repo to confuse developers

**Review by silentsakky — APPROVED** [review]

File: WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js (hunk 1 above)
Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: Actually i didn’t know about this at that point.

**Comment by ghen2** [comment]
This change broke display for certain messages, see https://forums.zimbra.org/viewtopic.php?f=15&t=69009

**Review by ghen2 — COMMENTED** [review]
This change broke display for certain messages, see https://forums.zimbra.org/viewtopic.php?f=15&t=69009

File: WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js (hunk 1 above)
Comment (ghen2) [reply WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js:167]: This change reintroduced the issue previously fixed by https://github.com/Zimbra/zm-web-client/pull/560, please rollback.

**Comment by madhavdhuppe** [comment]
> This change broke display for certain messages, see https://forums.zimbra.org/viewtopic.php?f=15&t=69009

Will be looking into this @ghen2

**Comment by ghen2** [comment]
Basically the issue fixed by https://github.com/Zimbra/zm-web-client/pull/560 (ZCS-8165) was reintroduced here, because that commit was partially reverted.  Reverting this change again fixes it for me.

**Comment by madhavdhuppe** [comment]
> Basically the issue fixed by #560 (ZCS-8165) was reintroduced here, because that commit was partially reverted. Reverting this change again fixes it for me.

Reverting ZCS-8165 will introduce this issue need to add fix for these two issues.

**Comment by ghen2** [comment]
I did not mean reverting ZCS-8165, but reverting this pull https://github.com/Zimbra/zm-web-client/pull/611.

**Pull request merged**